CMakeLists.txt

```diff
@@ -71,6 +71,7 @@ set(PROJECT_APP_DIR      ${CMAKE_CURRENT_SOURCE_DIR}/apps)
 set(PROJECT_BINDINGS_DIR ${CMAKE_CURRENT_SOURCE_DIR}/bindings)
 
 add_subdirectory(src)
+link_openmp()
```

Comment:
We have to call `target_link_libraries` after the target is created. Not pretty, but I'd prefer to handle any refactoring in a separate issue. This fixes the linking errors in https://github.com/DARMA-tasking/vt/pull/2399.